Sources/String+Sweetness.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public extension String {
+    public var isValidEmail: Bool {
+        let emailRegEx = ".*@.*"
```

Comment:
It doesn't have to be the previous one, but at least it should check for the current cases -> https://github.com/UseSweet/SweetFoundation/pull/16

Either that or it should check for whatever the backend accepts.